lib/recaptcha/helpers.rb

```diff
@@ -271,6 +271,7 @@ def self.recaptcha_v3_default_callback_name(action)
     private_class_method def self.default_callback(options = {})
       nonce = options[:nonce]
       nonce_attr = " nonce='#{nonce}'" if nonce
+      selector_attr = options[:id] ? "##{options[:id]}" : ".g-recaptcha"
```

Comment:
@grosser I'm now using the `id` directly. 

Alternatively we could decouple it so an id would only be used if you also pass in a `selector` matching the id, but I guess that would just over-complicate things.